### PR TITLE
Correct rico client usage, version set according to the other projects

### PR DESCRIPTION
**Pull request by giftkugel**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
   ],
   "dependencies": {
     "angular": "1.6.9",
-    "rico-js": "https://github.com/rico-project/rico-js.git#master",
+    "rico-js": "1.0.0-CR.3",
```
Comment (madmas) [inline bower.json:36]: as rico.js is not published to npm, I expect this to fail. 
Shouldn't this be somthing like "https://github.com/rico-project/rico-js/tree/1.0.0-CR.3" ?

**Review by madmas — COMMENTED** [review]
Please see comment on dependecy reference

File: bower.json (hunk 1 above)
Comment (giftkugel) [reply bower.json:36]: No. Bower will fetch the correct version because of the tags aka releases on github.

Comment (giftkugel) [reply bower.json:36]: https://github.com/rico-project/rico-js/releases

Comment (madmas) [reply bower.json:36]: allright, my misunderstanding :)

**Pull request merged**